packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -178,7 +180,7 @@ export default function FeedItemComponent({
   );
 
   switch (item.type) {
-    case 'post': {
+    case FeedItemType.RegularPost: {
```

Comment:
Why the `regular` here and for `ad` do we have other types?

Reply:
Lint issues: https://github.com/dailydotdev/apps/pull/3118#discussion_r1609656253